src/physics/euler.cpp

```diff
@@ -1166,7 +1166,7 @@ ::boundary_pressure_outflow (
         for (int istate=0; istate<nstate; ++istate) {
             soln_bc[istate] = soln_int[istate];
         }
-    } 
+    }
```

Comment:
avoid committing changes where its just a space or added empty line

Reply:
Sure, I'll reset this to how it is in master.